### PR TITLE
Feat/eth mainnet deployment

### PR DESCRIPTION
**Pull request by carlomazzaferro**
## Type of change

- [ ] Docs change / dependency upgrade
- [ ] Configuration / tooling changes
- [ ] Refactoring
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires changes in customer code

## Related Issue(s)

Fixes

### CODE REVIEW

File: packages/sdk/chainDeploy/mainnets/ethereum.ts [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+  uniswap: {
+    hardcoded: [],
+    uniswapData: [],
+    pairInitHashCode: ethers.utils.hexlify("0x96e8ac4277198ff8b6f785478aa9a39f403cb768dd02cbee326c3e7da348845f"),
+    uniswapV2RouterAddress: "0x68b3465833fb72A70ecDF485E0e4C7bD8665Fc45",
+    uniswapV2FactoryAddress: "0x5C69bEe701ef814a2B6a3EDD4B1652CB9cc5aA6f",
+    uniswapV3FactoryAddress: "0x1F98431c8aD98523631AE4a59f267346ea31F984",
+    uniswapOracleInitialDeployTokens: [],
+    uniswapOracleLpTokens: [],
+    flashSwapFee: 25,
```
Comment (vminkov) [inline packages/sdk/chainDeploy/mainnets/ethereum.ts:29]: looks like the fee is 30 https://docs.uniswap.org/contracts/v2/guides/smart-contract-integration/using-flash-swaps

**Review by vminkov — CHANGES_REQUESTED** [review]

File: packages/chains/src/ethereum/assets.ts [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+    name: "Governance OHM",
+    decimals: 18,
+    oracle: OracleTypes.UniswapTwapPriceOracleV2,
+    extraDocs: defaultDocs("https://etherscan.io", GOHM),
+    disabled: true,
+  },
+  {
+    symbol: assetSymbols.USDC,
+    underlying: USDC,
+    name: "USD Coin (Arb1)",
```
[the remark this answers is not in the document]
Comment (carlomazzaferro) [reply packages/chains/src/ethereum/assets.ts:57]: Nope, typo/artifact from copypasting

File: packages/chains/src/ethereum/index.ts [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+import fundingStrategies from "./fundingStrategies";
+import irms from "./irms";
+import liquidationDefaults from "./liquidation";
+import oracles from "./oracles";
+import specificParams from "./params";
+import deployedPlugins from "./plugins";
+import redemptionStrategies from "./redemptionStrategies";
+
+const chainConfig: ChainConfig = {
+  chainId: SupportedChains.arbitrum,
```
[the remark this answers is not in the document]
Comment (carlomazzaferro) [reply packages/chains/src/ethereum/index.ts:16]: nope, good catch

File: packages/sdk/chainDeploy/mainnets/ethereum.ts (hunk 1 above)
Comment (carlomazzaferro) [reply packages/sdk/chainDeploy/mainnets/ethereum.ts:29]: 👍

File: packages/chains/src/ethereum/index.ts (hunk 3 above)
[the remark this answers is not in the document]
Comment (vminkov) [reply packages/chains/src/ethereum/index.ts]: this is still arbitrum

File: packages/chains/src/ethereum/assets.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (carlomazzaferro) [reply packages/chains/src/ethereum/assets.ts]: ok now it is gone

File: packages/chains/src/ethereum/addresses.ts [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+import { UNISWAP_V3_ADDRESSES } from "../common/addresses";
+
+import { assets } from "./assets";
+
+const chainAddresses: ChainAddresses = {
+  W_TOKEN: underlying(assets, assetSymbols.WETH),
+  W_TOKEN_USD_CHAINLINK_PRICE_FEED: "0x5f4eC3Df9cbd43714FE2740f5E3616155c5b8419",
+  UNISWAP_V2_ROUTER: "0x68b3465833fb72A70ecDF485E0e4C7bD8665Fc45",
+  UNISWAP_V2_FACTORY: "0x5C69bEe701ef814a2B6a3EDD4B1652CB9cc5aA6f",
```
[the remark this answers is not in the document]
Comment (carlomazzaferro) [reply packages/chains/src/ethereum/addresses.ts:12]: I've corrected them

**Review by vminkov — APPROVED** [review]

**Pull request merged**